starter-project-web-vue2/components/ruth/RuthBlog.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div>
+    <nuxt-link
+      style="text-decoration: none; color: inherit"
+      :to="'/ruth/' + blog._id"
```

Comment:
nit; this could be 
:to="`/ruth/${blog._id}`"

Reply:
Okay, done.